picture-ts/src/lib/ui.ts

```
@@ -0,0 +1,461 @@
+
+                    if (this.showTimeElapsed) {
+                        payload.elapsed = formatTime(elapsed);
+                    }
+
+                    this.progressBar.update(current, payload);
+                }
+                break;
+            case 'simple':
+                let status = `\r${this.title}: ${percentage}% complete`;
+
+                if (this.showTokensPerSecond) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)